scout/server/blueprints/dashboard/controllers.py

```diff
@@ -23,130 +23,143 @@ def get_dashboard_info(adapter, institute_id=None, slice_query=None):
         },
         {
             'title': 'Cohort tag',
-            'count': general_info['cohort_cases'],
-            'percent': general_info['cohort_cases'] / total_cases,
+            'count': general_sliced_info['cohort_cases'],
+            'percent': general_sliced_info['cohort_cases'] / total_sliced_cases,
         }
     ]
-    if nr_validation_cases:
-        overview.append(
-        {
-            'title': 'Validation ordered',
-            'count': nr_validation_cases,
-            'percent': nr_validation_cases / total_cases,
-        })
-        overview.append(
-        {
-            'title': 'Validated',
-            'count': len(validated_cases),
-            'percent': len(validated_cases) / nr_validation_cases,
-        })
+
+    # Data from "Variant statistics tab" is not filtered by slice_query and numbers will
+    # reflect verified variants in all available cases for an institute
+    general_info = get_general_case_info(adapter, institute_id=institute_id)
+    total_cases = general_info['total_cases']
+    sliced_case_ids = general_sliced_info['case_ids']
+    verified_query = {
+        'verb' : 'validate',
+    }
+    if institute_id: # filter by institute if users wishes so
+        verified_query['institute'] =  institute_id
+
+    # Case level information
+    sliced_validation_cases = set()
+    sliced_validated_cases = set()
+
+    # Variant level information
+    validated_tp = set()
+    validated_fp = set()
+    var_valid_orders = 0 # use this counter to count 'True Positive', 'False positive' and 'Not validated' vars
+
+    validate_events = adapter.event_collection.find(verified_query)
+    for validate_event in list(validate_events):
+        case_id = validate_event.get('case')
+        var_obj = adapter.variant(case_id=case_id, document_id=validate_event['variant_id'])
+
+        if var_obj: # Don't take into account variants which have been removed from db
+            var_valid_orders += 1
+            if case_id in sliced_case_ids:
+                sliced_validation_cases.add(case_id) # add to the set. Can't add same id twice since it'a a set
+
+            validation = var_obj.get('validation')
+            if validation and validation in ['True positive', 'False positive']:
+                if case_id in sliced_case_ids:
+                    sliced_validated_cases.add(case_id)
+                if validation == 'True positive':
+                    validated_tp.add(var_obj['_id'])
+                elif validation == 'False positive':
+                    validated_fp.add(var_obj['_id'])
+
+    n_validation_cases = len(sliced_validation_cases)
+    n_validated_cases = len(sliced_validated_cases)
+
+    # append
+    overview.append(
+    {
+        'title': 'Validation ordered',
+        'count': n_validation_cases,
+        'percent': n_validation_cases / total_sliced_cases,
+    })
+    overview.append(
+    {
+        'title': 'Validated cases',
```

Comment:
Hm, might be misunderstood as nr of true positives. How about 'Validated cases (TP + FP)'?